### DISCUSSION TITLE
legacy boot

### DISCUSSION THREAD

**Discussion by mmud393** [sebanc/brunch#728] [Q&A]
i have a systen dell optiplex 745 with legacy boot is it support ??

**Comment by sebanc** [comment]
It should work, after extracting the main brunch archive, extract the mbr_support.tar.gz patch found in the master branch of this repo in the same folder (overwriting files when requested) and follow the rest of the instructions.

If the default kernel (5.4) does not work, press "e" when you see grub and change /kernel in the grub configuration with /kernel-4.19.

**Reply by simplyrajatgupta** [reply]
It may or may not work but this will work 
Bet me with ₹100 INR
[https://github.com/sebanc/brunch/issues/1088]()

**Comment by simplyrajatgupta** [comment]
Try this
[https://github.com/sebanc/brunch/issues/1088]()
